picard/album.py

```diff
@@ -261,24 +261,26 @@ def _finalize_loading(self, error):
 
         if not self._requests:
             # Prepare parser for user's script
-            if config.setting["enable_tagger_script"]:
-                script = config.setting["tagger_script"]
-                if script:
-                    parser = ScriptParser()
-                    for track in self._new_tracks:
-                        # Run tagger script for each track
-                        try:
-                            parser.eval(script, track.metadata)
-                        except:
-                            self.error_append(traceback.format_exc())
-                        # Strip leading/trailing whitespace
-                        track.metadata.strip_whitespace()
-                    # Run tagger script for the album itself
-                    try:
-                        parser.eval(script, self._new_metadata)
-                    except:
-                        self.error_append(traceback.format_exc())
-                    self._new_metadata.strip_whitespace()
+            if config.setting["enable_tagger_scripts"]:
+                for item in config.setting["list_of_scripts"]:
+                    if item[2]:
+                        script = item[3]
+                        if script:
```

Comment:
I reallly dislike `item[2]`& `item[3]`, what are they ? Iterating through the list of scripts should be simplified.